Library/Checkout.aspx.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Library.Data;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace Library
+{
+    public partial class Checkout : System.Web.UI.Page
+    {
+        int bookId = 0;
+        protected void Page_Load(object sender, EventArgs e)
+        {
+            if (!int.TryParse(Request.QueryString["ID"], out bookId))
+            {
+                Response.Redirect("~/BooksList.aspx");
+            }
+            if (!IsPostBack)
+            {
+                DataTable dataTable = DatabaseHelper.Retrieve(@"
+                    select p.FirstName + ' ' + p.LastName as Name, p.LibraryCardNumber, li.EmployeeNumber as EmployeeNumber
+                    from Librarian li
+                    join Patron p on p.LibraryCardNumber=li.LibraryCardNumber
+                ");
+                LibrarianDropDownList.DataTextField = "Name";
+                LibrarianDropDownList.DataValueField = "EmployeeNumber";
+                LibrarianDropDownList.AppendDataBoundItems = true;
+                LibrarianDropDownList.DataSource = dataTable;
+                LibrarianDropDownList.DataBind();
+
+                DataTable dt = DatabaseHelper.Retrieve(@"
+                    select LibraryCardNumber, FirstName + ' ' + LastName as PatronName
+                    from Patron
+                    order by PatronName
+                    ");
+                PatronDropDownList.DataTextField = "PatronName";
+                PatronDropDownList.DataValueField = "LibraryCardNumber";
+                PatronDropDownList.AppendDataBoundItems = true;
+                PatronDropDownList.DataSource = dt;
+                PatronDropDownList.DataBind();
+
+                DataTable data = DatabaseHelper.Retrieve(@"
```

Comment:
Rename the `dataTable`, `dt`, and `data` variables to something more meaningful.